### DISCUSSION TITLE
Numpy mappings

### DISCUSSION THREAD

**Discussion by asilvas** [josdejong/mathjs#2751] [Ideas]
Would be mighty handy to have a doc (or update existing [functions doc](http://mathjs.org/docs/reference/functions.html)) that shows the functional equivalents of Numpy, being it's so widely used. I get that not everything is a one-to-one, or that some stuff is even possible, but it would go a long ways towards helping those like myself trying to port over large or complex Python code. As an added benefit, it might also bring to light some useful missing capabilities to make this great library all the better.

**Comment by josdejong** [comment]
Thanks for your suggestion Aaron, such a tutorial and/or reference document helping people coming from python to see how things work in mathjs can be very helpful indeed. Maybe the same holds for people coming from Matlab/Octave.

Anyone interested in picking this up?
